draft-ietf-quic-transport.md

```diff
@@ -3293,7 +3293,7 @@ frame.  Its format is identical to the PATH_CHALLENGE frame
 
 If the content of a PATH_RESPONSE frame does not match the content of a
 PATH_CHALLENGE frame previously sent by the endpoint, the endpoint MAY generate
-a connection error of type UNSOLICITED_PATH_RESPONSE.
+a connection error of type PROTOCOL_VIOLATION.
```

Comment:
Should this specify the frame type as PATH_RESPONSE now that frame type is present in CONNECTION_CLOSE?

Reply:
Yes, but I didn't think that it would be necessary to state that explicitly.  If we decide to do that, I'd want to do that everywhere.

Reply:
Agreed, we might want to state it everywhere.